### DISCUSSION TITLE
Feel Free to post any issues or questions

### DISCUSSION THREAD

**Discussion by IN2-Moist** [IN2-Moist/2Take1-Moist-Script#1] [Q&A]
Feel Free to post any issues or questions & help for use of this script

**Comment by YouRan4** [comment]
Can I initiate an HTTP request in the script

**Reply by IN2-Moist** [reply]
Sadly no, something i been, brainstorming on! Using powershell or lua external console i hope to bridge this some how i have yet to find a way i can actually get to work. other than having it write to a file and execute from a line read the current menu api prevents this directly for security reasons

**Reply by Bluscream** [reply]
Easy with my ipc [tool](https://github.com/Bluscream/file-ipc) and script

**Comment by jzz0909** [comment]
can you add language options   :>

**Reply by IN2-Moist** [reply]
Im in the process of rewritting the script from scratch will look at adding possibility of this being scripted in the next gen release

**Reply by IN2-Moist** [reply]
Script for translation is released with the current release editing instructions are supplied in the readme

**Comment by MadMaxLIS** [comment]
Hi Moist,

I have suggestion if it's possible, add Acid Lab for "Set Stat" to be possible use R* Created Logo or R* Verified Logo.
Thanks for answer. Have a nice day.

MadMaxLIS

**Reply by IN2-Moist** [reply]
@MadMaxLIS 
If it can be changed in game to a custom name i guess it would be possible. I will  have to dig through the game stats. Will try to remember this when i am off work next in a few days and take a look

**Reply by MadMaxLIS** [reply]
Yes it was possible use example this &#166 ; AcidLab but R* patched it for all CEO office, MC Biker. Only your option "Set Stat" works.
Okay no rush needed, i hope you will remember on me. Stay safe and enjoy days off relax :)

**Reply by IN2-Moist** [reply]
done
![image](https://user-images.githubusercontent.com/4621091/215169193-7197e07f-648d-498b-a7e0-6be0f343dc17.png)
@MadMaxLIS

**Reply by MadMaxLIS** [reply]
@IN2-Moist Wow you did it ! really appreciate your work :)

**Reply by IN2-Moist** [reply]
@MadMaxLIS i pushed an unreleased update to the repo, its not fully tested i did write some code just now to address some small things that were annoying me. but it has the stuff in there for this request so feel free to test it and if you find any issues let me know. just ignore the out of date notifys when starting it, its in future release so its gonna say  it XD
